### PR TITLE
Header

### PR DESCRIPTION
**Pull request by goonerDp**
Header basic component. For now without sticky state.

### CODE REVIEW

File: docs/autocomplete.txt [hunk 1]
```diff
@@ -1,10 +1,4 @@
-Throttling and debouncing in JavaScript
-https://codeburst.io/throttling-and-debouncing-in-javascript-646d076d0a44
-
-google places api
-вводим в текстовое поле 
-ищем место по этим данным
-не вызывать поиск чаще 300мс
```
Comment (Red02Raccoon) [inline docs/autocomplete.txt:1]: hide key

Comment (goonerDp) [reply docs/autocomplete.txt:1]: done, the new one in google docs

**Comment by alexeenko-frontend** [comment]
Wrap index.scss file in folder ("styles").

**Comment by alexeenko-frontend** [comment]
Add right padding to autocomplete.

**Comment by alexeenko-frontend** [comment]
deepinscreenshot_ - _20181129162849
Different file naming style.

**Review by alexeenko-frontend — COMMENTED** [review]
No complaints about the code.

**Pull request closed without merging**